docs/sidebars.ts

```diff
@@ -30,8 +30,8 @@ const sidebars: SidebarsConfig = {
 
     {
       type: "category",
-      label: "Code Documentation",
-      items: [{ type: "autogenerated", dirName: "auto-docs" }],
+      label: "Schema Documentation",
```

Comment:
1. You need to add auto-schema, not replace auto-docs.
2. Add the Schema category